pkg/k8shandler/curation.go

```diff
@@ -255,7 +254,7 @@ func updateCuratorIfRequired(desired *batch.CronJob) (err error) {
 	current := desired.DeepCopy()
 
 	if err = sdk.Get(current); err != nil {
-		if apierrors.IsNotFound(err) {
+		if errors.IsNotFound(err) {
```

Comment:
where does `errors` come from now if not `"k8s.io/apimachinery/pkg/api/errors"`?

Reply:
we had that import twice... once named as `apierrors` and once as `k8s.io/apimachinery/pkg/api/errors`